vault/testing.go

```diff
@@ -24,6 +24,7 @@ import (
 	"path/filepath"
 	"sync"
 	"sync/atomic"
+	testing2 "testing"
```

Comment:
I believe it is because `testing` is already used by `"github.com/mitchellh/go-testing-interface"`. I think I can remove this import though. (It was added automatically when I did a Refactor -> Move.)